### DISCUSSION TITLE
devlife/2023-07-05-SSAFY-SSAFY-10%EA%B8%B0-%ED%95%A9%EA%B2%A9-%ED%9B%84%EA%B8%B0/

### DISCUSSION THREAD

**Discussion by giscus** [suakang17/suakang17.github.io#4] [Comments]
# devlife/2023-07-05-SSAFY-SSAFY-10%EA%B8%B0-%ED%95%A9%EA%B2%A9-%ED%9B%84%EA%B8%B0/

어느새 막학기를 마치고 다음달이면 졸업이다! 하지만 입사할 때 어필할만한 완성된 프로젝트 경험이 부족한 것 같아서 싸피에 지원했고 현재 10기로 다니는 중이다. 특히 내 목표는 외국계인데 레주메에 쓸만한 만족스러운 프로젝트가 하나도 없다 ㅜㅜ.

https://suakang17.github.io/devlife/2023-07-05-SSAFY-SSAFY-10%EA%B8%B0-%ED%95%A9%EA%B2%A9-%ED%9B%84%EA%B8%B0/

**Comment by gkswns3708** [comment]
갓갓갓

**Reply by suakang17** [reply]
갓준 👍
